fizz_buzz/fizzBuzzQuiz.js

```diff
@@ -0,0 +1,18 @@
+function fizzBuzz(max){
+	for(i=0; i<=max; i++){
+  	if(i % 3 == 0 && i % 5 == 0){
+    	console.log("fizzbuzz");
+    }
+    else if(i % 3 == 0){
+    	console.log("fizz");
+    }
+    else if(i % 5 == 0){
+    	console.log("buzz");
+    }
+    else{
+    	console.log(i);
+    }
+  };
+}
+
+fizzBuzz(99);
```

Comment:
this file should be moved into a VishakhaSethi directory. `/fizz_buzz/VishakhaSethi/fizzBuzzQuiz.js`